tests/model_registry/test_model_registry_creation.py

```diff
@@ -32,28 +31,51 @@ class TestModelRegistryCreation:
     # TODO: Switch to Python client
     @pytest.mark.smoke
     def test_registering_model(
-        self: Self,
-        model_registry_instance_rest_endpoint: str,
-        current_client_token: str,
-        updated_dsc_component_state,
+        self: Self, model_registry_instance_rest_endpoint: str, current_client_token: str, updated_dsc_component_state
     ):
-        cmd = generate_register_model_command(
-            endpoint=model_registry_instance_rest_endpoint, token=current_client_token
+        # address and port need to be split in the client instantiation
+        server = model_registry_instance_rest_endpoint.split(":")[0]
+        port = model_registry_instance_rest_endpoint.split(":")[1]
```

Comment:
```suggestion
        server, port = model_registry_instance_rest_endpoint.split(":")
```
can assign both directly